applications/asset_tracker/README.rst

```diff
@@ -220,22 +246,7 @@ Alternatively, use the command line tool ``menuconfig`` or configure the options
 .. external_antenna_note_end
 
 This application supports the |NCS| :ref:`ug_bootloader`, but it is disabled by default.
-To enable the immutable bootloader, set ``CONFIG_SECURE_BOOT=y``.
-
-
-Using nRF Cloud A-GPS or P-GPS
```

Comment:
@peknis I noticed just now that when I did this PR, I removed this section of the readme.  But now I have no idea why I removed it -- I think it's important information still.

Also, @peknis, at the time I did this PR, everyone decided that a changelog entry was not needed.  But since then, I see that two other ATv1 changes are mentioned in the change log.  So I am wondering if I should also add a change log entry now for this feature, prior to NCS 1.8.0 final.

Reply:
@plskeggs, you may want to bring the section back to the doc and also add an entry to the changelog. Please, open a PR and mark it with the 1.8.0 milestone if you want to have those changes in the upcoming release. If you decide to open a PR, add also @greg-fer as one of the reviewers as he is responsible for the doc release.

Reply:
False alarm -- that section was duplicated, so this change was just to remove one of them.  No need for a new PR.